frontend/packages/knative-plugin/src/actions/traffic-splitting.ts

```diff
@@ -0,0 +1,19 @@
+import { KebabOption } from '@console/internal/components/utils';
+import { setTrafficDistributionModal } from '../components/modals';
+
+export const setTrafficDistribution = (kind, obj): KebabOption => {
+  return {
+    label: 'Set Traffic Distribution',
+    callback: () =>
+      setTrafficDistributionModal({
+        obj,
+      }),
+    accessReview: {
+      group: kind.apiGroup,
```

Comment:
is `kind` a model here as we are accessing `kind.apiGroup` if yes then argument name as kind could be confusing

Reply:
changed the name to avoid confusion